server/management/commands/import_data.py

```diff
@@ -191,6 +206,15 @@ def handle(self, *args, **options):
                     not_in_india = True
                     state_ut = None
 
+                if not minors:
+                    occupation = clean_occupation(row[columns["occupation"]])
+                    if occupation in OCCUPATIONS:
```

Comment:
This check could go into the `clean_occupations` function to simplify the code here.

Reply:
moved it